packages/fluentui/docs/src/prototypes/FormValidation/FormHooks.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Form } from '@fluentui/react-northstar';
+import { useForm } from 'react-hook-form';
+
+const FormValidateHooks = () => {
+  const { register, handleSubmit, errors, setValue, watch, triggerValidation } = useForm({
+    mode: 'onBlur',
+  });
+
+  const onSubmit = data => {
+    console.log('errors: ', errors);
+    console.log('data: ', data);
+  };
+
+  const firstNameWatched = watch('firstName');
```

Comment:
I am not familiar with `react-hook-form`, but this breaks perf of the whole form as renders on each change:

![image](https://user-images.githubusercontent.com/14183168/85864774-3f7fc580-b7c5-11ea-9bf3-f832bee94da1.png)

Is there any chance to avoid it?

Reply:
removed